scripts/build/builders/host.py

```diff
@@ -88,6 +89,8 @@ def ExamplePath(self):
             return 'shell/standalone'
         elif self == HostApp.OTA_PROVIDER:
             return 'ota-provider-app/linux'
+        elif self == HostApp.SIMULATED_APP1:
+            return 'placeholder/linux/'
```

Comment:
how does one differentiate between app1 and app2 given that the path is just placehoder/linux?

Reply:
Never mind, I see zap config in gn build args.